browser/src/canvas/sections/ShapeHandlesSection.ts

```diff
@@ -102,9 +103,21 @@ class ShapeHandlesSection extends CanvasSectionObject {
 		this.convertToTileTwipsIfNeeded();
 		this.getHandles();
 		this.updateSize();
-		this.addSubSections();
-		this.sectionProperties.shapeRectangleProperties = this.getShapeRectangleProperties();
-		this.calculateInitialAnglesOfShapeHandlers();
+
+		if (GraphicSelection.extraInfo.isMathObject === false) {
```

Comment:
is `extraInfo` always present?

Reply:
`isMathObject` is a new property, to keep correct behavior even without core patch, we should check if it is present and true instead